docs/api/public_php_api.md

```diff
@@ -1125,6 +1125,20 @@ try {
 }
 ```
 
+##### delete translation from all versions
+
+To delete a translation from all Content item versions, use the `deleteTranslation` method on `ContentService`.
+
+```
+public function deleteTranslation(ContentInfo $contentInfo, $languageCode);
+```
+
+This operation is permanent, so the user should be warned about that.
+Since it is designed for maintenance tasks and might be a long-running operation,
+it should be used by a console command, not in the back office.
```

Comment:
Hmm, since I wrote this many months ago we've changed approach. So while to be safe I kept that in internal doc, we don't need to put it here as deleteTranslation is designed to have an UI (both in v1 and v2)